help/en/html/tools/throttle/BasicThrottleWindow.shtml

```diff
@@ -232,7 +232,7 @@
       The Throttle Window is Jynstrumentable, particularly with more advanced function
       controls.<br>
       <br>
-      <a href="https://jmri.org/jython/Jynstruments/ThrottleFrame/"> A few sample
+      <a href="https://www.jmri.org/jython/Jynstruments/ThrottleFrame/"> A few sample
       Jynstruments are delivered</a>, some for function control, others for camera image integratio
```

Comment:
While you are at it:
`..., others for camera image integratio`

The last word in the last sentense misses the last letter.